Packages/com.unity.inputsystem/InputSystem/Plugins/XR/XRTemplateBuilder.cs

```diff
@@ -117,6 +128,9 @@ internal static string OnFindLayoutForDevice(int deviceId, ref InputDeviceDescri
             InputSystem.RegisterLayoutBuilder(() => layout.Build(), layoutName, matchedLayout);
 
             return layoutName;
+#else //UNITY_INPUT_SYSTEM_ENABLE_XR
+            return null;
```

Comment:
Shouldn't hook itself in if it's not doing something.

Reply:
That makes sense, will adjust